strangerstudios/paid-memberships-pro
# Expand allowed HTML tags for pmpro_kses and add a way to stop normal processing

### All Submissions:






### Changes proposed in this Pull Request:

This allows additional HTML tags/attributes to be used in email templates.

For more complicated templating, someone will have to disable the kses processing using this new filter snippet we can provide: https://gist.github.com/sc0ttkclark/7297d8e8a41a2ac2309af778364e86da

### How to test the changes in this Pull Request:

1. Try saving an email template with `<!doctype html>` and see that it gets removed.
2. Enable the filter and save with `<!doctype html>` and see that it no longer gets removed.

### Other information:




### Changelog entry

> Support more complicated email templates with the new `pmpro_kses_pre` filter to override the `pmpro_kses` processing.
